doc/lua_api.txt

```diff
@@ -2732,6 +2735,20 @@ and `minetest.auth_reload` call the authetification handler.
     * spread these updates to neighbours and can cause a cascade
       of nodes to fall.
 
+### Mod channels
+You can find mod channels communication scheme in `docs/mod channels.png`.
+
+* `minetest.mod_channel_join(channel_name)`
+    * Server joins channel `channel_name`. If channel has no member it's created. You
+      should listen from incoming messages with `minetest.register_on_modchannel_message`
+      call to receive incoming messages
+* `minetest.mod_channel_leave(channel_name)`
+    * Server leaves channel `channel_name`. No more incoming or outgoing messages can be
+      sent to this channel from server mods.
+* `minetest.mod_channel_send(channel_name, message)`
```

Comment:
needs more documentation:

Are channels reliable/ordered? Or something else.

Also, what about sending a message to a single client?

Reply:
mod channels are not for sending message to a single client, a channel is a hub, if a client wants a private channel with server should join a private channel with him too. It's a CSM and SSM contract between both mods.